### PR TITLE
Update S3 integration SDK documentation for multi-file input

### PR DESCRIPTION
**Pull request by cw75**

### CODE REVIEW

File: api-reference/sdk-reference/package-aqueduct/package-aqueduct.integrations/aqueduct.integrations.s3_integration.md [hunk 1]
```diff
@@ -25,27 +25,39 @@ Class for S3 integration.
...
 ```
 
-Retrieves a file from the S3 integration.
+Reads multiple files from the S3 integration into a single TableArtifact.
 
 **Arguments**:
 
-  filepath:
-  Filepath to retrieve from.
+  filepaths:
```
Comment (vsreekanti) [inline api-reference/sdk-reference/package-aqueduct/package-aqueduct.integrations/aqueduct.integrations.s3_integration.md:38]: Is this documentation reflected in the SDK? If not, we should update it there and autogenerate these docs.

**Review by vsreekanti — COMMENTED** [review]
@cw75, these docs should be autogenerated -- if they're not, let's update the SDK docs first and then autogenerate these. We also need to update the documentation under "Using Integrations".

**Comment by cw75** [comment]
@vsreekanti updated.

**Comment by vsreekanti** [comment]
Great, thanks @cw75! We need to also update the guide to using S3 to reflect these changes as well: https://docs.aqueducthq.com/integrations/using-integrations/aws-s3

**Comment by cw75** [comment]
@vsreekanti updated.

File: integrations/using-integrations/aws-s3.md [hunk 2]
```diff
@@ -12,10 +12,18 @@ my_bucket = client.integration('s3/my_data_bucket')
...
+* `format`: The expected table format when reading from `filepaths`. We currently support `CSV`, `Parquet`, or `JSON`.
+
+See below for example usage:
 
 ```python
-customers = my_bucket.file('data/sales/customers.csv')
+customers = my_bucket.file(filepaths='data/sales/customers.csv', format='CSV')
+
+# When retrieving multiple tables, we concatenate them into a single DataFrame.
+cars = my_bucket.file(filepaths=['data/car/mazda.json', 'data/car/honda.json'], format='JSON')
```
Comment (vsreekanti) [inline integrations/using-integrations/aws-s3.md:26]: English nit: cars, not car.

File: integrations/using-integrations/aws-s3.md [hunk 3]
```diff
@@ -12,10 +12,18 @@ my_bucket = client.integration('s3/my_data_bucket')
...
+* `filepaths`: One or more paths from which files should be read.
+* `format`: The expected table format when reading from `filepaths`. We currently support `CSV`, `Parquet`, or `JSON`.
+
+See below for example usage:
 
 ```python
-customers = my_bucket.file('data/sales/customers.csv')
+customers = my_bucket.file(filepaths='data/sales/customers.csv', format='CSV')
+
+# When retrieving multiple tables, we concatenate them into a single DataFrame.
```
Comment (vsreekanti) [inline integrations/using-integrations/aws-s3.md:25]: Let's explicitly specify this in the text above as well. Using `s3.file`, you can retrieve either one file or many files. If it's many files, we concat into a DF. 

We're also missing how prefix search works here, I believe.

File: integrations/using-integrations/aws-s3.md [hunk 4]
```diff
@@ -33,7 +41,5 @@ Aqueduct does not eagerly validate whether or not your file exists. In order to
 In order to save an aritfact to a database, you can call `.save()` on the artifact. `.save()` expects information about which system to write the artifact to and what configuratin parameters are necessary. This configuration can be generated by calling `.config` on your integration object. For AWS S3, the following parameters are required:
 
 * `filepath`: The path in the S3 bucket to which this file should be written.
-* `format`: `csv`, `parquet`, or `json`
-  * `csv`: The Artifact will be converted to a CSV file (using the Pandas `to_csv` function) before writing it to `filpath`&#x20;
```
Comment (vsreekanti) [inline integrations/using-integrations/aws-s3.md:37]: Why was this removed?

**Review by vsreekanti — APPROVED** [review]
Thanks @cw75! Still a couple missing bits -- see comments inline. Should be good to merge once you fill the gaps in.

File: integrations/using-integrations/aws-s3.md (hunk 4 above)
Comment (cw75) [reply integrations/using-integrations/aws-s3.md:37]: The original version seems unnecessarily detailed.

Comment (vsreekanti) [reply integrations/using-integrations/aws-s3.md:37]: IMO, I don't think detail in documentation is a bad thing, unless it's misleading or inaccurate.

File: integrations/using-integrations/aws-s3.md (hunk 3 above)
Comment (cw75) [reply integrations/using-integrations/aws-s3.md]: updated

File: integrations/using-integrations/aws-s3.md (hunk 4 above)
Comment (cw75) [reply integrations/using-integrations/aws-s3.md:37]: Things like `to_csv ` is an implementation detail, which I think we don't have to reveal here for the user? No strong opinion though if you want it back.

**Comment by cw75** [comment]
@vsreekanti updated again

**Review by vsreekanti — APPROVED** [review]
Looks good!
